### DISCUSSION TITLE
FLOSS Weekly interview?

### DISCUSSION THREAD

**Discussion by jp-bennett** [e-m-b-a/emba#1240] [General]
Hey, EMBA looks really slick. Love that it's Open Source, too. I'm the host of FLOSS Weekly, one of the oldest podcasts about Open Source. Would one or two of the project leads like to come talk about it on the podcast? Respond here or email at floss at hackaday dot com. Let me know!

Jonathan Bennett

**Comment by m-1-k-3** [comment]
Hi Jonathan,

 thanks for reaching us out. I will ping you next week via email.
Have a good weekend.

Best,
Mike

**Reply by jp-bennett** [reply]
Hoping we can make this work. As of right now we have the 30th open. See https://docs.google.com/spreadsheets/d/1znEmwiDRjujujRaL_AmJbZgahG2z9EQuLs8C3U64wcI/edit?gid=0#gid=0

**Comment by m-1-k-3** [comment]
Thanks to the great work from @jp-bennett our FLOSS weekly session is already online. Check it out here: https://hackaday.com/2024/09/25/floss-weekly-episode-802-emba-layers-upon-layers-of-bash/

Thanks for this great show ... we had a lot funn
